### PR TITLE
DO-273: Added to functionality to show confirmation popup whenever us…

### PR DESCRIPTION
**Pull request by dhruvlahoti**
…er tries to delete the family member.

### CODE REVIEW

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+  const renderConfirmPopup = () => {
+    if (showConfirmPopup) {
+      return (
+        <Modal
+          isCloseButtonHidden
+          type={MODAL_TYPES.CONFIRM}
+          onClickSubmit={element.onDropIndexClick(
+            element.index)}
+          onClickCancel={closeDeleteConfirmModal}
+          submitButtonText="Yes, Delete"
```
Comment (ayushmalik2110) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx:24]: Use this from constant file. We have constant for button text.

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx [hunk 2]
```diff
@@ -1,17 +1,45 @@
...
+          onClickCloseModal={closeDeleteConfirmModal}
+        >
+          <div>Do you want to delete this detail?</div>
```
Comment (ayushmalik2110) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx:27]: Use this text from  constant.json

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx (hunk 1 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx]: Done

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx (hunk 2 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateRemoveButton/ArrayTemplateRemoveButton.jsx:31]: Done

**Pull request merged**